Sources/SwiftCBOR/CBOR.swift

```diff
@@ -19,23 +19,24 @@ public indirect enum CBOR : Equatable, Hashable,
     case double(Float64)
     case `break`
 
-    public var hashValue : Int {
+    public func hash(into hasher: inout Hasher) {
```

Comment:
Is this for Swift 5? Does this break Swift 4?

Reply:
I think the `Hasher` changes were actually a part of Swift 4.2.

If you'd like I can leave the old code in and add a `#if swift(>=5.0)` and include the new code in that, just to be extra safe?